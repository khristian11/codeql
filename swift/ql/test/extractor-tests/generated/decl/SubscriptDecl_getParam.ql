import codeql.swift.elements
import TestUtils

from SubscriptDecl x, int index
where toBeTested(x)
select x, index, x.getParam(index)
