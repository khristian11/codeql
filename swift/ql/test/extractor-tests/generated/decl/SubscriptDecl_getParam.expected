| declarations.swift:104:3:109:3 | subscript ... | 0 | declarations.swift:104:13:104:16 | x |
| declarations.swift:111:3:113:3 | subscript ... | 0 | declarations.swift:111:13:111:16 | x |
| declarations.swift:111:3:113:3 | subscript ... | 1 | declarations.swift:111:21:111:25 | y |
